Emby.Server.Implementations/Data/SqliteItemRepository.cs

```diff
@@ -2753,10 +2739,10 @@ private void LogQueryTime(string methodName, string commandText, DateTime startD
         {
             var elapsed = (DateTime.UtcNow - startDate).TotalMilliseconds;
 
-            int slowThreshold = 1000;
+            int slowThreshold = 100;
 
 #if DEBUG
-            slowThreshold = 250;
+            slowThreshold = 10;
```

Comment:
These values should probably be documented and maybe even put in a central file.

Reply:
Before we document them, we should agree on the values

Reply:
The values make sense to me. It will produce a lot of warnings in DEBUG, but seeing as we want to optimize it anyway, it's fine with me.